db/migration/src/m20240226_000006_create_peer_store.rs

```diff
@@ -0,0 +1,90 @@
+use sea_orm_migration::prelude::*;
+use sea_orm_migration::sea_orm::prelude::ChronoDateTimeUtc;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
```

Comment:
So does this make the table also persistent if using the file sqlite connection? Because we need to figure out a reasonable strategy of forgetting the peers as well.

Reply:
Yes. We will need to prune the table on node start based on `LastSeen`